File: src/14.ts
import { promises as fs } from 'fs'

const solve14 = async (size: number): Promise<string> => {
    return (await fs.readFile('./res/popular-names.txt', 'utf-8'))
        .split('\n')
        .slice(0, size)
        .join('\n')
}

solve14(10).then(console.log)

File: src/14.py
# head -n 10 res/popular-names.txt

from helper.file_helper import read_string

source = read_string("./res/popular-names.txt")


def solve_14(size: int) -> str:
    return "\n".join(source.split("\n")[:size])


print(solve_14(10))
